subcommittees/mentoring/README.md

```diff
@@ -1,73 +1,86 @@
 
-## What We Do 
+## What We Do
 
 ### Instructor Discussion Sessions
-Each week, we host four instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
 
-To find an upcoming session, visit this link: http://pad.software-carpentry.org/instructor-discussion
+Each week, volunteers host two instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
 
-### Instructor-Helper Retreat
-In November 2015, Software Carpentry and Data Carpentry instructors and helpers got together together at sites around the world for a day of sharing skills, trying out new lessons and ideas and discussing all things instructor and helper related. 
+To find an upcoming session, visit this link: [http://pad.software-carpentry.org/instructor-discussion][discussion]
 
-To see the schedule for the 2015 retreat, visit: http://swcarpentry.github.io/instructor-retreat-2015/
+### Mentoring Program
 
-### Archived: Pre- and Post- Workshop Debriefing Session
+This program supports instructors who are new to our community by matching them with a personal Mentor. Mentors will guide small groups (~4-5) of Mentees, holding meetings for their Mentees to do things like practice teaching, ask questions about workshop logistics, and brush up on lesson content, depending on their needs and interests. 
 
-Before launching the Instructor Discussion Sessions, we used to host separate [pre- and post-workshop debriefing sessions](http://software-carpentry.org/blog/categories/#debriefing). We are grateful to all the hosts and participants who attended these sessions.
+See [this page](https://github.com/datacarpentry/mentoring-program/blob/master/program-outline.md) for a full outline of the program.  
 
-## Mailing List
+## Mentoring Committee Meetings
 
-The best way to contact this subcommittee is via the [mailing
-list][mailing-list], which is archived [here][mailing-list-archives].
+The mentoring committee itself meets regularly to evaluate current activities, and propose
+new activities.  Minutes of past meetings are available [here](minutes).  All community 
+members are welcome to attend mentoring committee meetings.  Similarly, we welcome 
+anyone who is interested in joining up and helping out!  
 
-## Outreach
+To attend our next meeting, visit this link: [http://pad.software-carpentry.org/scf-mentoring][meetings]
```

Comment:
I'd rather [avoid putting URLs in the link text][1] here as well.

[1]: https://github.com/swcarpentry/board/pull/175/files#r103553558